js added, created footer and skills

A-Kaliexe opened the pull request:
I did as suggested for the side navigation menu overlapping. I removed the position: fixed property from the .sidebar-nav class and took off left-margin as well, for the overlap.  I adjusted the sidebar's width and content alignment in the media queries as well.

A-Kaliexe committed:
js added, created footer and skills

toma63 reviewed (changes requested):
looks much better and great overall.  I noticed that when first loaded, some of the words are cut off in the sidebar.  This is probably caused by inconsistent and conflicting width settings across the default, hover and media queries.  I'd also recommend using em units (character widths) rather than px.  I tried changing everything to 6em or 12 em and it fixes the issue.

A-Kaliexe committed:
Fix sidebar responsiveness and hover; update media queries

A-Kaliexe commented:
Thank you. I did as suggested and made the px to em.  Is it okay if I decrease the em sizes I used .sidebar-nav{ 9.5em} instead of 12. I added from your suggested changes to the media queries as well.
 2. Should I change everything in px to em to keep the styling consistent, excluding the nav bar that has been adjusted to it?
 3.I’ve also updated the media queries to address the icon-only view.  Would it be better to remove the hover effect if there are still issues?
